backend/cognito/template.yaml

```diff
@@ -0,0 +1,494 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  Cognito UserPool and Client
+Transform:
+  - AWS::Serverless-2016-10-31
+
+Resources:
+  UserPool:
+    Type: AWS::Cognito::UserPool
+    Properties:
+      Policies:
+        PasswordPolicy:
+          MinimumLength: 6
+          RequireUppercase: false
+          RequireLowercase: false
+          RequireNumbers: false
+          RequireSymbols: false
+          TemporaryPasswordValidityDays: 7
+      Schema:
+        - Name: sub
+          AttributeDataType: String
+          DeveloperOnlyAttribute: false
+          Mutable: false
+          Required: true
+          StringAttributeConstraints:
+            MinLength: "1"
+            MaxLength: "2048"
+        - Name: email
+          AttributeDataType: String
+          DeveloperOnlyAttribute: false
+          Mutable: true
+          Required: false
+          StringAttributeConstraints:
+            MinLength: "0"
+            MaxLength: "2048"
+        - Name: phone_number
+          AttributeDataType: String
+          DeveloperOnlyAttribute: false
+          Mutable: true
+          Required: false
+          StringAttributeConstraints:
+            MinLength: "0"
+            MaxLength: "2048"
+      AutoVerifiedAttributes:
+        - email
+      UsernameAttributes:
+        - email
+      SmsVerificationMessage: '{####} is your GOV.UK Sign In security code.'
+      EmailVerificationMessage: |-
```

Comment:
Can the HTML blocks be stored separately and referenced in this template?

Reply:
Apparently not: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cognito-userpool.html#cfn-cognito-userpool-emailverificationmessage

Other options include smooching stuff together before applying CloudFormation like we did with the Canaries for Verify.  However, that's not a job for this PR in my opinion - these is just getting something together so we can create repeatable environments and do pipeline stuff (and do manual deploys whilst the pipeline happens).